cpp/tests/prims/mg_per_v_transform_reduce_dst_key_aggregated_outgoing_e.cu

```diff
@@ -0,0 +1,597 @@
+                  sg_dst_key.view(),
+                  sg_kv_store.view(),
+                  key_aggregated_e_op_t<vertex_t, weight_t, result_t>{},
+                  property_initial_value,
+                  cugraph::reduce_op::elementwise_minimum<result_t>{},
+                  cugraph::get_dataframe_buffer_begin(global_result));
+              } else {
+                per_v_transform_reduce_dst_key_aggregated_outgoing_e(
+                  *handle_,
+                  sg_graph_view,
+                  sg_src_prop.view(),
+                  cugraph::edge_dummy_property_t{}.view(),
+                  sg_dst_key.view(),
+                  sg_kv_store.view(),
+                  key_aggregated_e_op_t<vertex_t, thrust::nullopt_t, result_t>{},
+                  property_initial_value,
+                  cugraph::reduce_op::elementwise_minimum<result_t>{},
+                  cugraph::get_dataframe_buffer_begin(global_result));
+              }
+              break;
+            case reduction_type_t::ELEMWISE_MAX:
+              if (sg_edge_weight_view) {
+                per_v_transform_reduce_dst_key_aggregated_outgoing_e(
+                  *handle_,
+                  sg_graph_view,
+                  sg_src_prop.view(),
+                  *sg_edge_weight_view,
+                  sg_dst_key.view(),
+                  sg_kv_store.view(),
+                  key_aggregated_e_op_t<vertex_t, weight_t, result_t>{},
+                  property_initial_value,
+                  cugraph::reduce_op::elementwise_maximum<result_t>{},
+                  cugraph::get_dataframe_buffer_begin(global_result));
+              } else {
+                per_v_transform_reduce_dst_key_aggregated_outgoing_e(
+                  *handle_,
+                  sg_graph_view,
+                  sg_src_prop.view(),
+                  cugraph::edge_dummy_property_t{}.view(),
+                  sg_dst_key.view(),
+                  sg_kv_store.view(),
+                  key_aggregated_e_op_t<vertex_t, thrust::nullopt_t, result_t>{},
+                  property_initial_value,
+                  cugraph::reduce_op::elementwise_maximum<result_t>{},
+                  cugraph::get_dataframe_buffer_begin(global_result));
+              }
+              break;
+            default: FAIL() << "should not be reached.";
+          }
+
+          ASSERT_TRUE(compare(mg_aggregate_results, global_result));
+        }
+      }
+    }
+  }
+
+ private:
+  static std::unique_ptr<raft::handle_t> handle_;
+};
+
+template <typename input_usecase_t>
+std::unique_ptr<raft::handle_t>
+  Tests_MGPerVTransformReduceDstKeyAggregatedOutgoingE<input_usecase_t>::handle_ = nullptr;
+
+using Tests_MGPerVTransformReduceDstKeyAggregatedOutgoingE_File =
+  Tests_MGPerVTransformReduceDstKeyAggregatedOutgoingE<cugraph::test::File_Usecase>;
+using Tests_MGPerVTransformReduceDstKeyAggregatedOutgoingE_Rmat =
+  Tests_MGPerVTransformReduceDstKeyAggregatedOutgoingE<cugraph::test::Rmat_Usecase>;
+
+#if 0
```

Comment:
Is there a reason these are skipped?  Should we document that reason, eliminate the skipping, or eliminate the code?

Reply:
Yes, cugraph::kv_store_t does not compile with cuco mode and thrust tuple type value. I am planning to fix this in a separate PR but I will also add a FIXME here.